CHANGELOG.md

```diff
@@ -88,16 +107,10 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 * `jj restore` now supports `-i`/`--interactive` selection.
 
-* Add templater support for rendering commit signatures.
-
 * `jj file list` now supports templating.
 
-* `jj status` now shows untracked files when they reside directly under a tracked directory.
-
-* New `merge-tools.<TOOL>.diff-expected-exit-codes` config option to suppress
-  warnings from tools exiting with non-zero exit codes.
-
-* Add a new template alias `builtin_op_log_oneline` along with `format_operation_oneline` and `format_snapshot_operation_oneline`
+* Add a new template alias `builtin_op_log_oneline` along with
+  `format_operation_oneline` and `format_snapshot_operation_oneline`
```

Comment:
nit: The imperative mood here (and on lines 128-134) feels out of place with the rest of the items. We're not telling the reader to add the template alias.

I'm also not sure it's clear to the reader what this means. Maybe something like:

> There is a new `builtin_op_log_oneline` template alias you can pass to `jj op log -T` for a more compact output. You can use  `format_operation_oneline` and `format_snapshot_operation_oneline` to customize parts of it.

Reply:
Updated, thanks.